config/locales/en.yml

```diff
@@ -1,6 +1,78 @@
 # Sample localization file for English. Add more files in this directory for other locales.
 # See http://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
-en: 
+en:
```

Comment:
Looks like default archive config uses en as well, so if we change it there, tests should pick it up. Have we agreed which one will be the default going forward for live?